src/OutputManager.php

```diff
@@ -41,10 +40,10 @@ public function __construct(ConfigInterface $config)
     /**
      * Saves the service and types php code to file
      *
-     * @param PhpClass $service
+     * @param ZendClassGenerator $service
      * @param array $types
      */
-    public function save(PhpClass $service, array $types)
+    public function save($service, array $types)
```

Comment:
Why not keep just change the typehint from `Wsdl2PhpGenerator\PhpSource\PhpClass` to `Zend\Code\Generator\ClassGenerator`?